Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Util.cpp

```diff
@@ -24,6 +26,18 @@ int CLuaFunctionDefs::GetValidPedModels(lua_State* luaVM)
         }
     }
 
+    // gather our, engineRequestModel skins
+    // start at 313, so we wont reinsert the same skin ids(sometimes 3 is allocated, and that would appear twice in the list if we would start at 0)
+    for (const CClientModel* model : m_pManager->GetModelManager()->GetModelsByType(eClientModelType::CCLIENTMODELPED, 313)) 
+    {
+        lua_pushnumber(luaVM, ++iIndex);
+        lua_pushnumber(luaVM, model->GetModelID());
+        lua_settable(luaVM, -3);
+    }
+
+    // Another option could be to loop thru all MAX_MODELS skins,
```

Comment:
That comment doesn't really fit in here, just stating it in a pull request is perfectly fine